frontend/public/features.ts

```diff
@@ -4,7 +4,15 @@ import { connect } from 'react-redux';
 import { Map as ImmutableMap } from 'immutable';
```

Comment:
@dtaylor113 Do you mind rolling back the changes to this file since it was part of @rhamilto's PR? The diff moves things around, but doesn't add anything.